lib/openstudio-standards/version.rb

```diff
@@ -14,4 +14,4 @@ def self.git_revision
     return 'git-not-found-on-this-system'
   end
   VERSION = '0.2.15'.freeze
-end
\ No newline at end of file
+end
```

Comment:
We typically ignore the autocorrect changes made by rubocop to `version.rb`, `Weather.Model.rb`, and `Weather.stat_file.rb`. The diff shown in this PR for these files seem reasonable, @mdahlhausen - could we accept these changes? Do you recall the reason why we usually exclude these changes?

Reply:
@lymereJ the rubocop edits are fine.  I'm surprised some method/file endings got changed - maybe worth a doublecheck that we have what rubocop wants.